less-builder.js

```diff
@@ -20,33 +36,63 @@ var cssInject = [
 ].join(';');
 
 exports.bundle = function (loads, compileOpts, outputOpts) {
-	var _this = this;
+    var loader = this;
+    var writeStubs = typeof outputOpts == 'undefined';
+    outputOpts = outputOpts || compileOpts;
+
+    var stubDefines = writeStubs ? loads.map(function(load) {
+      return (compileOpts.systemGlobal || 'System') + ".register('" + load.name + "', [], false, function() {});";
```

Comment:
See @olostan, he's concatenating `(compileOpts.systemGlobal || 'System')` with the remaining strings. This is what I mentioned in #2 discussion.